### ISSUE TITLE
Rework Recorder to not depend on the Defects4j validation script

### ISSUE THREAD

**Issue by Marti2203** [nus-apr/cerberus#29]
Similarly to #22, Recorder is mostly tailored for a specific benchmark. This is not too hard to do but will require investigation into the scripts of recorder and making a whole new docker image probably.

**Comment by rshariffdeen** [comment]
@Marti2203  - I believe you were able to make progress in this ?

**Comment by Marti2203** [comment]
Yes, I will add it this weekend. I found the script so will make some updates.
